.solidarity

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json.schemastore.org/solidaritySchema",
+  "requirements": {
+    "Node": [
+      {
+        "rule": "cli",
+        "binary": "node",
+        "semver": "^10.16.0"
+      }
+    ],
+    "Yarn": [
+      {
+        "rule": "cli",
+        "binary": "yarn",
+        "version": "--version",
+        "semver": "^1.17.3"
+      }
+    ],
+    "Cocoapods and Fastlane": [
+      {
+        "rule": "cli",
+        "binary": "ruby",
+        "semver": "2.5.1",
+        "error": "Ruby is required to run Cocoapods and Fastlane. We recommend installing Ruby with a version manager (https://github.com/echobind/react-native-template-echobind#prerequisites)"
+      },
+      {
+        "rule": "cli",
+        "binary": "bundle",
+        "error": "Bundler is used to install RubyGems"
+      },
+      {
+        "rule": "cli",
+        "binary": "pod",
+        "semver": "1.7.1",
+        "error": "Cocopods is required to be installed as of React Native 0.60"
+      },
+      {
+        "rule": "cli",
+        "binary": "convert",
```

Comment:
Should this be `gm` instead of `convert`? Going based off the [docs](http://www.graphicsmagick.org/convert.html)
```suggestion
        "binary": "gm",
```

Reply:
works for me. both are installed I believe.